CHANGELOG.md

```diff
@@ -716,6 +716,15 @@ Other minor changes
          → dsubst₂ C p q (f x₁ y₁) ≡ f x₂ y₂
   ```
 
+* Added vector associativity proof to 
+  `Data/Vec/Relation/Binary/Equality/Setoid.agda`:
+  ```
+  ++-assoc : ∀ {n m k} (xs : Vec A n) → (ys : Vec A m) 
+      → (zs : Vec A k) → (xs ++ ys) ++ zs ≋ xs ++ (ys ++ zs)
+  ++-assoc [] ys zs = ≋-refl
+  ++-assoc (x ∷ xs) ys zs = refl ∷ ++-assoc xs ys zs
```

Comment:
We don't tend to add implementations to the CHANGELOG. For consistency's sake, any chance you could remove it?